sdk/src/main/java/com/oursky/authgear/latte/LatteLinkActivity.kt

```diff
@@ -0,0 +1,18 @@
+package com.oursky.authgear.latte
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+
+class LatteLinkActivity : AppCompatActivity() {
```

Comment:
I suppose the usage is to add this activity to app's AndroidManifest with appropriate intent filter.

Looks like it we need to provide a theme to ensure UI is not flashed while handling the link.